Scraper/scraper.py

```
@@ -0,0 +1,41 @@
+import requests
+
+def fetch_book_data(book_name):
+    """
+    Fetch metadata for a book using Google Books API
+    """
+
+    url = f"https://www.googleapis.com/books/v1/volumes?q={book_name}"
+
+    try:
+        response = requests.get(url)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)